### ISSUE TITLE
Problems with HTML5 validation and Dropzone

### ISSUE THREAD

**Issue by wernerkrauss** [bigfork/silverstripe-dropzone#48]
Hi, I wanted to make my Image upload with Dropzone and also make it a required field by adding `->setAttribute('required', true)`.

Unfortunately this blocks HTML5 validation and I cannot send the form.

Is there any chance to get this field required, with HTML5 validation if possible?

**Comment by lozcalver** [comment]
I think there are two problems with HTML5 validation:

1. Using `RequiredFields` doesn’t add the `required="required"` attribute to the `<input />` automatically like it should
2. The `<input />` is hidden, so the HTML5 validation message never shows

Looks like you’ve found a workaround for 1, and I think you should be able to fix 2 in your own CSS until I get round to fixing it in the module. Something like:

```css
div.dropzonefield .form__field-holder {
    position: relative;
}

div.dropzonefield input {
    position: absolute;
    display: block;
    inset: 0;
    opacity: 0;
    pointer-events: none;
} 
```

Basically instead of hiding the `<input />`, just make it invisible instead.
